### ISSUE TITLE
Can You give us any ideas on how to improve Accuracy of it.

### ISSUE THREAD

**Issue by raymax0x** [RamBagga/LinearRegressionModel#2]
I am Beginner in ML. Would be grateful if you can help out.

**Comment by RamBagga** [comment]
Try changing number of epochs , play around with the learning rate to find the most efficient one also you can try learning different ML algos which suit this problem more(other than gradient descent) !
